docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -518,7 +518,7 @@ controllers but does require that the KubeStellar Core Space (KCS) has been set
 An example usage follows.
 
 ```shell
-kubectl kubestellar ensure location --imw root:imw-1 demo1 foo=bar the-word=the-bird
+kubectl kubestellar ensure location --imw imw-1 demo1 foo=bar the-word=the-bird
```

Comment:
The definitional line above that says `--imw` takes a "workspace_path" also needs to change. I expect this is not the only place such a change needs to be made.

Before this PR, this document talked plainly and specifically about kcp workspaces. This PR changes some of the commands to deal with Space object names. There is a choice to be made about this document now: do we revise the whole thing to eradicate all references to kcp workspaces and instead talk about Space objects? Even if we did, it would still be necessary to document the relation between Space object names and kcp workspace pathnames because kcp is still the provider being used. I think that the introductory material of this document needs to be expanded to state that both Space objects and kcp workspaces are in play and state the relation between Space object names and kcp workspace pathnames.

Reply:
Please see https://github.com/ezrasilvera/edge-mc/blob/sfdocs/space-framework/docs/space-framework.md#kcp-spa
explaining the relations between Space names and KCP's workspace name and workspace path.

Reply:
My point is that `docs/content/Coding Milestones/PoC2023q1/commands.md` needs to be maintained.